darts/models/rnn_model.py

```diff
@@ -139,6 +150,9 @@ def __init__(self,
             training. Generally speaking, `training_length` should have a higher value than `input_chunk_length`
             because otherwise during training the RNN is never run for as many iterations as it will during
             training. For more information on this parameter, please see `darts.utils.data.ShiftedDataset`
+        likelihood_model
+            Optionally, the likelihood model to be used for probabilistic forecasts.
```

Comment:
Maybe you could say here it expects a `darts.utils.LikelihoodModel` so that users reading the docstring know where to look to instantiate their likelihood.